main.py

```diff
@@ -0,0 +1,84 @@
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.button import Button, ButtonBehavior
+from kivy.uix.image import Image
+from kivy.clock import Clock
+import random
+from kivy.core.audio import SoundLoader
+from kivy.app import App
+from kivy.uix.screenmanager import ScreenManager, Screen
+
+# Declaring screens for use in screen manager
+class BattleScreen(Screen):
+    def check_for_monster(self):
+        try:
+            return self.monster
+        except:
```

Comment:
Not sure if this is an appropriate way of using try/except.
Especially since no specific exception is specified. So there could be an exception that isn't caused by the player not having stats yet, but the program will still behave the same.